crm_claim_type/views/crm_claim.xml

```diff
@@ -14,4 +14,31 @@
             </xpath>
         </field>
     </record>
+
+    <record id="crm_claim_type_search_view" model="ir.ui.view">
+        <field name="name">CRM - Claims Search</field>
+        <field name="model">crm.claim</field>
+        <field name="inherit_id" ref="crm_claim.view_crm_case_claims_filter" />
+        <field name="arch" type="xml">
+            <xpath expr="//search" position="inside">
+                <filter
+                    string="Claim Type"
+                    name="claim_type"
+                    domain="[('claim_type', '!=', False)]"
+                />
+            </xpath>
+            <xpath expr="//group" position="inside">
+                <filter
+                    name="claim_type_group"
+                    string="Claim Type"
+                    context="{'group_by': 'claim_type'}"
+                />
+                <filter
```

Comment:
This part shouldn't be in this module, as the stage is not added by `crm_claim_type`.